### PR TITLE
treewide: replace pythonImportCheck(s) with pythonImportsCheck

### LINKED ISSUE #340809: typo : pythonImportCheck instead of pythonImportsCheck
### Describe the bug

4 packages have the `pythonImportCheck` attribute, which is a typo for `pythonImportsCheck`

### Steps To Reproduce
1. `rg pythonImportCheck`
2. see 4 packages:

```
pkgs/development/python-modules/pyfunctional/default.nix
37:  pythonImportCheck = "pyfunctional";

pkgs/development/python-modules/python-ffmpeg/default.nix
22:  pythonImportCheck = [ "ffmpeg" ];

pkgs/development/python-modules/pyairports/default.nix
22:  pythonImportChecks = [ "pyairports" ];

pkgs/development/python-modules/geoparquet/default.nix
37:  pythonImportCheck = "geoparquet";
```

### Expected behavior

no result

### Notify maintainers

- [ ] pyfunctional & geoparquet: @Sigmanificient
- [ ] python_ffmpeg: @roshaen 
- [ ] pyairports: @cfhammill 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by eclairevoyant**
## Description of changes

fixes #340809

- **python312Packages.geoparquet: fix typo**
- **python312Packages.pyaiports: fix typo**
- **python312Packages.pyfunctional: fix typo**
- **python312Packages.python-ffmpeg: fix typo**

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by eclairevoyant** [comment]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python311Packages.pyfunctional</li>
    <li>python311Packages.pyfunctional.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.pyfunctional</li>
    <li>python312Packages.pyfunctional.dist</li>
    <li>python312Packages.vllm</li>
    <li>python312Packages.vllm.dist</li>
  </ul>
</details>
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python311Packages.geoparquet</li>
    <li>python311Packages.geoparquet.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.pyairports</li>
    <li>python311Packages.pyairports.dist</li>
    <li>python311Packages.python-ffmpeg</li>
    <li>python311Packages.python-ffmpeg.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.pyairports</li>
    <li>python312Packages.pyairports.dist</li>
    <li>python312Packages.python-ffmpeg</li>
    <li>python312Packages.python-ffmpeg.dist</li>
  </ul>
</details>


```
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: pyfunctional
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pyfunctional'
```

```
No ROCm runtime is found, using ROCM_HOME='/nix/store/1w7sl37p0xrf6i7m4jzwd6nyqhbg5a8l-clr-6.0.2'
<string>:65: RuntimeWarning: Failed to embed commit hash:
[Errno 2] No such file or directory: 'git'
Traceback (most recent call last):
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    main()
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    json_out['return_val'] = hook(**hook_input['kwargs'])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/513rk7p8n17499jglnavq46pkz5k1ab3-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_>
    return hook(config_settings)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 327,>
    return self._get_build_requires(config_settings, requirements=[])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 297,>
    self.run_setup()
  File "/nix/store/llgbdr264r5ih0d19ii7k5c1ankxh9sv-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 313,>
    exec(code, locals())
  File "<string>", line 472, in <module>
  File "<string>", line 393, in get_vllm_version
RuntimeError: Unknown runtime environment

ERROR Backend subprocess exited when trying to invoke get_requires_for_build_wheel
```
```

File: pkgs/development/python-modules/pyfunctional/default.nix [hunk 1]
```diff
@@ -34,7 +34,7 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [ pytestCheckHook ];
-  pythonImportCheck = "pyfunctional";
+  pythonImportsCheck = "pyfunctional";
```
Comment (nim65s) [inline pkgs/development/python-modules/pyfunctional/default.nix:37]:
```suggestion
  pythonImportsCheck = "functional";
```

**Comment by nim65s** [comment]
the vllm issue should be fixed by https://github.com/NixOS/nixpkgs/pull/340321 I guess

**Comment by eclairevoyant** [comment]
> the vllm issue should be fixed by #340321 I guess, but it seems to require a pytorch rebuild to check that, so I'm not going to do that right now.

Thanks for pointing that out; it's not strictly related to this PR anyway, so I'll not worry about it.

**Comment by eclairevoyant** [comment]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.vllm</li>
    <li>python312Packages.vllm.dist</li>
  </ul>
</details>
<details>
  <summary>20 packages built:</summary>
  <ul>
    <li>python311Packages.geoparquet</li>
    <li>python311Packages.geoparquet.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.pyairports</li>
    <li>python311Packages.pyairports.dist</li>
    <li>python311Packages.pyfunctional</li>
    <li>python311Packages.pyfunctional.dist</li>
    <li>python311Packages.python-ffmpeg</li>
    <li>python311Packages.python-ffmpeg.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.pyairports</li>
    <li>python312Packages.pyairports.dist</li>
    <li>python312Packages.pyfunctional</li>
    <li>python312Packages.pyfunctional.dist</li>
    <li>python312Packages.python-ffmpeg</li>
    <li>python312Packages.python-ffmpeg.dist</li>
  </ul>
</details>

**Review by nim65s — APPROVED** [review]

**Review by youhaveme9 — APPROVED** [review]

**Comment by Sigmanificient** [comment]
Made the typo...twice 😭

**Review by Sigmanificient — APPROVED** [review]

**Pull request merged**